Lists/scamlinks.txt

```diff
@@ -241,6 +241,7 @@ dicsord-gives.com
 dicsord-hypesquads.com
 dicsord-nitro.com
 dicsord-nitro.ru
+discord-gg.shop
```

Comment:
We don't accept contributions to `scamlinks.txt` anymore to avoid confusion. Please add the link to `scams.txt`
```suggestion

```